backend/options.go

```diff
@@ -0,0 +1,44 @@
+package backend
+
+import (
+	"time"
+
+	"github.com/relab/gorums"
+	"github.com/relab/hotstuff"
+)
+
+type backendOptions struct {
+	location          string
+	locationInfo      map[uint32]string
```

Comment:
This could be `map[hotstuff.ID]string` and similarly on Line 21. Then you could avoid the type cast on Line 23. May also require changes elsewhere, e.g., at `WithLatencyInfo` call sites.